File: java/javadpbo/src/com/sem4dpbo/week7/interfaceBangunDatar/Persegi.java
package com.sem4dpbo.week7.interfaceBangunDatar;

public class Persegi implements BangunDatar{

    Persegi(){}

    @Override
    public double luas(int s1, int s2) {
        return (s1 * s2);
    }

    @Override
    public double keliling(int s1, int s2) {
        return (2 * (s1 + s2));
    }
}

File: php/7th_week/interface_bangundatar/Persegi.php
<?php

class Persegi implements BangunDatar{

    function __construct()
    {
        
    }

    // method pada bangund datar wajib di implementaikan di dalam class

    function luas($s1, $s2)
    {
        return ($s1 * $s2);
    }

    function keliling($s1, $s2)
    {
        return 2 * ($s1 * $s2);
    }

    function __destruct()
    {
        
    }
}
?>